db/migrate/20180420032328_add_userstate_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserstateToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :excluded_at, :datetime
+  end
+end
```

Comment:
The term _state_ is too general. Programming is all about state. Let's be more specific with the terminology. Amend:
- the name of this migration `AddExcludedAtToUsers`
- the filename of this migration (will need to rollback before making changes)
- the name of this commit (I'll show you how to make an empty fixup commit to remind you of something)

Reply:
changes at:
cf6f4e5a6d00633cd578276c011cee0436d6debd